quickdialog/QEntryTableViewCell.m

```diff
@@ -140,7 +141,7 @@ - (void)prepareForElement:(QEntryElement *)element inTableView:(QuickDialogTable
     _textField.secureTextEntry = _entryElement.secureTextEntry;
     _textField.clearsOnBeginEditing = _entryElement.clearsOnBeginEditing;
     _textField.contentVerticalAlignment = UIControlContentVerticalAlignmentCenter;
-    _textField.textAlignment = _entryElement.appearance.entryAlignment;
+    _textField.textAlignment = NSTextAlignmentNatural;
```

Comment:
With this change, using appearance will not work anymore. You should make this change in QFlatAppearance (and potentially QClassicApperance, but it's not used anymore since iOS 7 I believe).